client/my-sites/stats/activity-log/index.jsx

```diff
@@ -4,7 +4,8 @@
 import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 import { localize } from 'i18n-calypso';
-import { groupBy, map, get } from 'lodash';
+import { groupBy, map, get, filter } from 'lodash';
+import moment from 'moment';
```

Comment:
As Timmy said, we don't need to import this, as we can get it from the Localize props